services/account_type/cloudtrail_owner.yaml

```diff
@@ -36,39 +36,15 @@ Resources:
         - Key: cloudzero-reactor-account-id
           Value: !Ref ReactorAccountId
 
-  SnsTopicPolicy:
-    Type: 'AWS::SNS::TopicPolicy'
-    Condition: CreateResources
-    Properties:
-      Topics:
-        - !Ref CloudTrailSNSTopicArn
-      PolicyDocument:
-        Version: '2012-10-17'
-        Statement:
-          - Sid: AWSCloudTrailSNSPolicyCloudZeroReactor20170207
-            Effect: Allow
-            Principal:
-              Service: cloudtrail.amazonaws.com
-            Action:
-              - 'sns:Publish'
-            Resource: !Ref CloudTrailSNSTopicArn
-          - Sid: AWSCloudTrailCrossAccountSNSPolicy20170207ForCloudZeroReactor
-            Effect: Allow
-            Principal:
-              AWS: !Sub 'arn:aws:iam::${ReactorAccountId}:root'
-            Action:
-              - 'sns:Subscribe'
-              - 'sns:Receive'
-              - 'sns:ListSubscriptionsByTopic'
-            Resource: !Ref CloudTrailSNSTopicArn
-
   SnsSubscription:
     Type: AWS::SNS::Subscription
     Condition: CreateResources
     Properties:
       Protocol: sqs
       Endpoint: !GetAtt SqsQueue.Arn
-      Region: !Ref "AWS::Region"
+      Region: !Select
```

Comment:
b/c this SNS Topic could be in another region, read the region from the Topic Arn